nextflow_schema.json

```diff
@@ -483,6 +489,74 @@
                 }
             }
         },
+        "locidex": {
+            "title": "Locidex",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "lx_min_evalue": {
+                    "type": "number",
+                    "default": 0.0001
+                },
+                "lx_min_dna_len": {
+                    "type": "integer",
+                    "default": 1
+                },
+                "lx_min_aa_len": {
+                    "type": "integer",
+                    "default": 1
+                },
+                "lx_max_dna_len": {
+                    "type": "integer",
+                    "default": 10000000
+                },
+                "lx_max_aa_len": {
+                    "type": "integer",
+                    "default": 10000000
+                },
+                "lx_min_dna_ident": {
+                    "type": "number",
+                    "default": 80
+                },
+                "lx_min_aa_ident": {
+                    "type": "number",
+                    "default": 80
+                },
+                "lx_min_dna_match_cov": {
+                    "type": "number",
+                    "default": 80
+                },
+                "lx_min_aa_match_cov": {
+                    "type": "number",
+                    "default": 80
+                },
+                "lx_max_target_seqs": {
+                    "type": "number",
+                    "default": 10
+                },
+                "lx_extraction_mode": {
+                    "type": "string",
+                    "default": "raw"
+                },
+                "lx_report_mode": {
+                    "type": "string",
+                    "default": "normal"
+                },
+                "lx_report_prop": {
+                    "type": "string",
+                    "default": "locus_name"
+                },
+                "lx_report_max_ambig": {
+                    "type": "number",
+                    "default": 0
+                },
+                "lx_report_max_stop": {
+                    "type": "number",
+                    "default": 0
+                }
```

Comment:
I know requirements might be different for parameters that are to be provided by users through Irida Next and those that are not expected to be, but I still think it would be good practice, where applicable, to have the following for all parameters:

- type
- default
- format
- pattern
- min/max
- description